aioircd.py

```diff
@@ -285,7 +284,23 @@ async def JOIN(self, args):
 
     @command
     async def PRIVMSG(self, args):
-        pass
+        if not args or args[0] == "":
+            raise ErrNoRecipient("PRIVMSG")
+        user,*mess = args
+        message = " ".join(mess)
+        if not message or not message.startswith(":") or len(message)<2:
+            raise ErrNoTextToSend()
+        if user.startswith("#"):
+            chann = self.local.channels.get(user)
+            if chann == None:
+                raise ErrNoSuchChannel(user)
+            await chann.send_all(f":{self.nick} PRIVMSG {user} :{message[1:]}")
+        else:
+            receiver = self.local.users.get(user)
+            if receiver == None:
+                raise ErrErroneusNickname(user)
+            await receiver.send(f":{self.nick} PRIVMSG {user} :{message[1:]}")
+        await self.send(f"301 {self.nick} :{message[1:]}")
```

Comment:
Do we plan to support the AWAY command ? Why do we send a away message every time ? Looks like a bug.